bitacross-worker/enclave-runtime/src/rpc/worker_api_direct.rs

```diff
@@ -49,8 +49,9 @@ use itp_top_pool_author::traits::AuthorApi;
 use itp_types::{DirectRequestStatus, RsaRequest, ShardIdentifier, H256};
 use itp_utils::{FromHexPrefixed, ToHexPrefixed};
 use jsonrpc_core::{serde_json::json, IoHandler, Params, Value};
+#[cfg(feature = "development")]
 use lc_scheduled_enclave::{ScheduledEnclaveUpdater, GLOBAL_SCHEDULED_ENCLAVE};
-use litentry_macros::if_not_production;
+use litentry_macros::if_development;
```

Comment:
How about moving this line
```
use lc_scheduled_enclave::{ScheduledEnclaveUpdater, GLOBAL_SCHEDULED_ENCLAVE};
```

into the `if_development!({` block below?

There might be multiple places.

Reply:
I tried using the macro with `use` statements but it does not compile in some cases

e.g:
```
error: expected item after attributes
  --> app-libs/stf/src/trusted_call.rs:76:1
   |
76 | / if_development!({
77 | |     use crate::helpers::ensure_enclave_signer_or_alice;
78 | | });
   | |__^ in this macro invocation
   |
   = note: this error originates in the macro `if_development` (in Nightly builds, run with -Z macro-backtrace for more info)

```